integration_tests/helpers/util.js

```diff
@@ -153,7 +153,7 @@ function consoleLogBalance(balance) {
     );
 }
 
-function consoleLogCoinbaseDetails(txnDetails) {
+function consoleLogTransactionDetailsFormat2(txnDetails) {
```

Comment:
Let's just use one formatter. I as a developer would not know in which case to use one or the other

Reply:
Yip